site/public/css/colors.css

```diff
@@ -205,200 +209,204 @@
-
-      --standard-pale-lime-green: #dff0d8;
-      --standard-light-lime-green: #c3e3b5;
-      --standard-medium-lime-green: #99cb84;
-      --standard-medium-dark-lime-green: #7dfa78;
-
-      --standard-vibrant-dark-blue: #006699;
-      --standard-bronze: #E6AC00;
-      --standard-pastel-pink: #f2dede;
-      --standard-light-pink: #ff9999;
-      --standard-tangerine-yellow: #ffcc00;
-      --standard-pastel-yellow: #fcf8e3;
-      --standard-pale-yellow: #ffff99;
-      --standard-light-yellow: #ffff66;
-      --standard-light-yellow-brown: #8E7306;
-      --standard-beige: rgb(119, 119, 106);
-      --standard-light-gold: #fff3cd;
-      --standard-light-green: #427742;
-      --standard-pastel-blue: #2f6580;
-      --standard-light-blue: #99ccff;
-      --standard-light-seafoam: #a4dad5;
-
-      /*the look of the webpage itself*/
-      --background-blue: #202225;
-      --main-body-white: #ffffff;
-      --subheading-underscore-grey: #888888;
-      --subtle-grey-background: #555555;
-
-      /*dropdown buttons*/
-      --hover-dropdown: #f5f5f5;
-      --disabled-dropdown: #888888;
-      --focus-dropdown: #f5f5f5;
-
-      /*text and fonts*/
-      --text-black: #ffffff; /*use only on white/background-blue*/
-      --text-white: #000000; /*use on all colors*/
-      --btn-text-white: #ffffff;
-      --btn-text-black: #000000;
-      --btn-default-text: #ffffff;
-
-      /*discussion forum*/
-      --viewed-discussion-post-blue: #e8f0f7;
-      --deleted-discussion-post-grey: #dcdcdc;
-      --important-post: #ffd700; /*is also for star color*/
-      --attachment-box-color: #4a4a4a;
-      --file-upload-table-hover: #4a4a4a;
-
-
-          /*these differ from the colors on the posts for some reason?
-          so I will put them here for reference
-          green-message: #008800
-          blue-message: #0a6496
-          red-message99: #af0000
-          yellow-message: #ff9900
-          black-message: #000000*/
-
-      /*links*/
-      --external-link-blue: #9B9BF3;
-      --external-link-hover: #a1a1ec;
-      --external-link-disabled: #888888;
-      --external-link-active: #991313;
-
-      /*sidebar*/
-      --selected-nav-sidebar: #535353;
-      --selected-nav-sidebar-text: #6c6c6c;
-      --nav-sidebar-text: #c8c8c8;;
-      --sidebar-hover-text: #ffffff;
-
-
-      /*tables*/
-      --table-info: #566268;
-
-      /*datepicker background colors*/
-      --date-picker-green: #446f44;
-      --date-picker-yellow: #9d9131;
-      --date-picker-red: #953d3d;
-
-      /* other */
-      --box-shadow-very-transparent: rgba(255,255,255,0);
-      --box-shadow-slightly-transparent: rgba(255,255,255,0);
-      --no-image-available: #ADADAD;
-      --standard-input-background: #4a4a4a;
-      --checkbox-black: #5296db;
-      --overall-comment: #4E5353;
-      --alternating-row-color-light: #4d4848;
-      --grading-note: #A8A8A8;
-      --file-browser-blue: #ffffff;
-      --extra-credit-blue: #2DACF0;
+    /*master color*/
```

Comment:
What are all these changes? If they're just spacing changes it should be reverted but scrolling quickly I don't see any real changes

Reply:
Looks like somehow I made a bunch of indentation changes. I'll undo them.